src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+
+	private String value;
```

Comment:
You might like to change the word value to something like block. Value is too general.